rasa/utils/tensorflow/models.py

```diff
@@ -103,6 +103,10 @@ def fit(
     ) -> None:
         """Fit model data"""
 
+        if not silent:
```

Comment:
Why do we need this if construct? Isn't fit() called only during training?
Also, if its necessary then we should rename silent to something more meaningful?

Reply:
It is also called during load, that's why I added the comment `# don't setup tensorboard writers when training during loading`. We can maybe rename it or add another parameter.

Reply:
Ahh, right its called in load to pass through a dummy batch.
Yes, maybe adding a separate parameter for it is better.